test/unit/preprocess/parameterproperty/ParameterPropertyHandlerRange-test.ts

```diff
@@ -23,17 +24,34 @@ function expectOutputProperties(output: Resource | undefined, expected: Resource
   }
 }
 
+// Copy of internal interface declaration in ParameterPropertyHandlerRange
+interface ILiteralTypeResult {
```

Comment:
Can't we just import it from the ParameterPropertyHandlerRange file?

Reply:
Yes, if you're ok with exporting this interface!

Reply:
Sure! We might want to rename it slightly though, to make more sense in a global context.

Reply:
e.g. `ILiteralAsTypeInterpretationResult`. But open for other suggestions.

Reply:
Check, sounds good to me!